tests/performance/common/common.go

```diff
@@ -46,8 +46,44 @@ const (
 	ServiceHandler    = "/api/v1/namespaces/default/services"
 	NodelabelKey      = "k8snode"
 	NodelabelVal      = "kb-perf-node"
+	ListContainer     = "/containers/json"
+	ContainerStats    = "/containers/%s/stats?stream=0" //stream – 1/True/true or 0/False/false, pull stats once then disconnect. Default true.
 )
 
+type ContainerList []struct {
+	ID     string   `json:"Id"`
+	Names  []string `json:"Names"`
+	Labels struct {
+		IoKubernetesContainerName string `json:"io.kubernetes.container.name"`
+		IoKubernetesPodName       string `json:"io.kubernetes.pod.name"`
+	} `json:"Labels"`
```

Comment:
Please remove this struct defination out of ContainerList, used as a field instead.